custom_components/octopus_energy/sensor.py

```diff
@@ -577,6 +579,20 @@ def setup_heat_pump_sensors(hass: HomeAssistant, account_id: str, heat_pump_id:
     entities.append(OctopusEnergyHeatPumpDataLastRetrieved(hass, coordinator, account_id, heat_pump_id))
 
   if heat_pump_response.octoHeatPumpControllerConfiguration is not None:
+    entities.append(OctopusEnergyHeatPumpSensorFixedTargetFlowTemperature(
+        hass,
+        coordinator,
+        heat_pump_id,
+        heat_pump_response.octoHeatPumpControllerConfiguration.heatPump
+      ))
+    
+    entities.append(OctopusEnergyHeatPumpNumberFixedTargetFlowTemperature(
```

Comment:
suggestion: This should be loaded as part of number.py

Reply:
This makes sense!